apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/concurrent/ConcurrentRdaSink.java

```diff
@@ -33,17 +121,55 @@
    */
   public ConcurrentRdaSink(
       int maxThreads, int batchSize, Supplier<RdaSink<TMessage, TClaim>> sinkFactory) {
```

Comment:
great work on making this so compact !

would be cool if we can add anything here to help future devs decipher the schedulers, publisher, processors...etc.